crates/mock/src/utils.rs

```diff
@@ -0,0 +1,238 @@
+use base64::engine::general_purpose;
+use base64::Engine;
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+use std::str::FromStr;
+
+use serde_json::{self};
+
+use solana_client::rpc_client::RpcClient;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::{account::Account, genesis_config::ClusterType};
+use yellowstone_grpc_proto::geyser::{SubscribeUpdateAccount, SubscribeUpdateAccountInfo};
+use yellowstone_vixen_core::{AccountUpdate, Parser};
+use yellowstone_vixen_parser::{
+    TokenExtensionProgramParser, TokenExtensionState, TokenProgramParser, TokenProgramState,
+};
+
+use crate::async_client::fetch_account_async;
+
+const FIXTURES_PATH: &str = "./fixtures";
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct AccountInfo {
+    pub data: Vec<String>,
+    pub executable: bool,
+    pub lamports: u64,
+    pub owner: String,
+    #[serde(rename = "rentEpoch")]
+    pub rent_epoch: u64,
+    pub space: u64,
+}
+
+pub async fn fetch_account_sync(pubkey: &Pubkey, cluster: ClusterType) -> Option<Account> {
```

Comment:
doesn't need to be async 
```suggestion
pub fn fetch_account_sync(pubkey: &Pubkey, cluster: ClusterType) -> Option<Account> {
```